tests/packages/Microsoft.Bot.Components.Telephony.Tests/Integration/BatchInputTests/BatchInput_FixedLengthBaseScenario.test.dialog

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "../../../tests.schema",
+  "$kind": "Microsoft.AdaptiveDialog",
+  "id": "BatchInput_BaseScenario.test",
+  "triggers": [
+    {
+      "$kind": "Microsoft.OnConversationUpdateActivity",
+      "actions": [
+        {
+          "$kind": "Microsoft.Telephony.FixedLengthBatchInput",
+          "property": "conversation.DTMFResult",
+          "prompt": "Enter your 6 digit phone number.",
+          "batchLength": "6"
+        },
+        {
+          "$kind": "Microsoft.SendActivity",
+          "activity": "${conversation.DTMFResult}"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "intent": "HelpIntent",
```

Comment:
Can the batched input be interrupted? Can we add a test to see if how help phrase will affect the flow?

Reply:
I'll add interrupt tests in a future PR